lib/sock_js/sock_js_utils.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:math';
+
+class SocketJsUtils
```

Comment:
This seem to be unused. I think that this should either be used automatically internally, or not be part of the library. Also the naming is probably a typo

Reply:
yes, it is intentional.
I can change if you want

Reply:
i changed name and i added unit test